typescript-selenium-webdriver/README.md

```diff
@@ -19,7 +19,8 @@ Some good places to start reading are:
 * [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver) to automate browsing to the page from the tests
 * [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs) to run an axe accessibility scan on the page from the Selenium browser
 * [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to run the tests in a CI build with every Pull Request
-* [axe-sarif-converter](https://github.com/microsoft/axe-sarif-converter) and [Sarif Viewer Build Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab) to visualize the results in Azure Pipelines
+* [axe-sarif-converter](https://github.com/microsoft/axe-sarif-converter) to convert axe results to SARIF format
+* Install [Sarif Viewer Build Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab) azure devops extension, to visualize the results in Azure Pipelines

```

Comment:
The "Install" before the link looks out of place compared to the rest of the bullets; everything else is a listing of what's being used, but this is an action for the user to take instead.

I think for the purposes of this PR, separating the bullets is probably fine. There's a different issue (#29) that asks for this list to be reworked and made a little less overwhelming; I think making the use/action for each bullet more approachable would be good to do as part of that other issue, but probably best to do it for all the bullets consistently rather than 1 at a time :)